pkg/console/operatorclient/operatorclient.go

```diff
@@ -0,0 +1,61 @@
+package operatorclient
+
+import (
+	"k8s.io/client-go/tools/cache"
+
+	operatorv1 "github.com/openshift/api/operator/v1"
+	operatorv1client "github.com/openshift/client-go/operator/clientset/versioned/typed/operator/v1"
+	operatorv1informers "github.com/openshift/client-go/operator/informers/externalversions"
+	"github.com/openshift/console-operator/pkg/api"
+)
+
+type OperatorClient struct {
+	Informers operatorv1informers.SharedInformerFactory
+	Client    operatorv1client.ConsolesGetter
+}
+
+func (c *OperatorClient) Informer() cache.SharedIndexInformer {
+	return c.Informers.Operator().V1().Consoles().Informer()
+}
+
+func (c *OperatorClient) GetOperatorState() (*operatorv1.OperatorSpec, *operatorv1.OperatorStatus, string, error) {
```

Comment:
I might have missed it, but I don't see where these functions are used

Reply:
I believe this is a file that is borrowed (ie, copy/paste) from other operators. I think they all impl basically the same methods, with minor tweaks to impl.  This is probably a good candidate for a resource that may be in `client-go` in the future, is my guess, at least the `interface`.